src/clients/python/sams_client/schemas/assets.py

```diff
@@ -39,14 +40,24 @@ class AssetStates(NamedTuple):
         'type': 'string',
         'mapping': not_analyzed
     },
-    '_created_by': {
+    'original_creator': {
         'type': 'string',
         'mapping': not_analyzed
     },
-    '_updated_by': {
+    'version_creator': {
         'type': 'string',
         'mapping': not_analyzed
     },
+    'firstcreated': {
+        'type': 'datetime',
+        'mapping': not_analyzed,
+        'default': utcnow()
+    },
+    'versioncreated': {
+        'type': 'datetime',
+        'mapping': not_analyzed,
+        'default': utcnow()
```

Comment:
This should be moved to the Set/Asset services (as setting it here will default to the datetime when the application starts).
We also don't want to import anything from `superdesk` in the client library, otherwise we have to add `superdesk-core` as a dependency to the client library.